.buildkite/pipeline.verify.yml

```diff
@@ -247,3 +247,10 @@ steps:
   - label: "Build docs :book:"
     command:
       - make build-docs
+
+  - label: ":firecracker: Firecracker RootFS"
+    command:
+      - packer init -upgrade firecracker/packer/build-rootfs.pkr.hcl
```

Comment:
I would incorporate this into the makefile target, rather than externalizing it like this.